src/app/tests/suites/certification/Test_TC_BI_1_1.yaml

```diff
@@ -28,34 +28,25 @@ tests:
               - name: "nodeId"
                 value: nodeId
 
-    - label: "read the global attribute: ClusterRevision"
+    - label: "Read the global attribute: ClusterRevision"
       command: "readAttribute"
       attribute: "ClusterRevision"
       response:
           value: 1
-
-    - label: "Read the global attribute constraints: ClusterRevision"
-      command: "readAttribute"
-      attribute: "ClusterRevision"
-      response:
           constraints:
               type: uint16
 
-    - label:
-          "write the default values to mandatory global attribute:
-          ClusterRevision"
-      command: "writeAttribute"
-      attribute: "ClusterRevision"
+    #Disabled due to issue #17483
+    - label: "Read the global attribute : FeatureMap"
```

Comment:
Why is this interactive?

Reply:
@tcarmelveilleux current yaml framework does not support this attribute read for Binary Input (Basic) cluster  .so we are using prompts to enable execution of this test in the harness

Reply:
It is not true that the YAML does not support it! It's just that the chip-tool has bugs. Let's be clear about that... The YAML operation exists, and should be using the correct method, as before, and be replaced by a disabled flag.

*NOTE THAT* if ppl are using chip-tool to read an attribute (to validate this and say "Yes"), then chip-tool needs to be able to read it, otherwise they have no tool. chip-tool and the YAML use the exact same method to read!

Reply:
> It is not true that the YAML does not support it! It's just that the chip-tool has bugs.

chip-tool does not have bugs here, fwiw.  It can read the attribute fine, both via YAML here and via the command line.

What has bugs is the server, which does not implement the attribute, because the spec for that changed recently and the server impls have not been updated to match that spec. So this test should be:
1. Enabled.
2. Skipped in CI _for now, not forever_ because the server apps we use in CI would fail this test.